eclair-core/src/test/scala/fr/acinq/eclair/channel/states/h/ClosingStateSpec.scala

```diff
@@ -722,6 +722,22 @@ class ClosingStateSpec extends TestkitBaseClass with StateTestsHelperMethods {
     relayerA.expectNoMsg(100 millis)
   }
 
+  test("recv BITCOIN_TX_CONFIRMED (remote commit, option_static_remotekey)", Tag("static_remotekey")) { f =>
```

Comment:
I think we also need the `next remote commit` case to be tested with static_remotekey.
Those tests are too important to risk missing something :)

Reply:
Good catch ;) done the test in 4abf88d